src/main/java/Counter.java

```diff
@@ -0,0 +1,19 @@
+public class Counter {
+    private int value;
+
+    public Counter() {
+        this.value = 0;
```

Comment:
It is unnecessary, 0 is default value for primitives

Reply:
done